src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -44,6 +47,24 @@ class ClaimSimulation extends Simulation {
     randomChar.toString
   }
 
+  private def addressStep =
+    if (features.getOrElse("ADDRESS_LOOKUP_ENABLED", false)) {
+      http("send postcode")
+        .post("/postcode")
+        .formParam("postcode", "AA1 1AA")
+        .formParam("_csrf", "${csrf_token2}")
+    }
+    else {
+      http("send_manual_address")
```

Comment:
Won't the manual address page always be present, so if their address cannot be found by the postcode lookup then they'll go to the manual address page? If so, I think this should be outside the else block and we only have the ADDRESS_LOOKUP_ENABLED toggle on the postcode lookup page.

Reply:
Updated so that postcode and manual address are entered if the lookup is enabled.

Reply:
Updated to go to manual address as well as postcode if enabled.